spec/results-view-spec.coffee

```diff
@@ -319,15 +319,17 @@ describe 'ResultsView', ->
         itemHeight = resultsView.find('.selected').outerHeight()
         pageHeight = Math.round(resultsView.innerHeight() / itemHeight) * itemHeight
         initialScrollTop = resultsView.scrollTop()
+        itemsPerPage = Math.floor(pageHeight / itemHeight)
+        originalItemIndex = Math.floor(initialScrollTop / itemHeight) + itemsPerPage
```

Comment:
I had a little trouble understanding what `originalItemIndex` represents. Is it the index of the selected item before `core:page-up` is triggered? If so, maybe we should just add an assertion that the `li` at that index is initially `selected`?